local-tests/setup/tinny-environment.ts

```diff
@@ -0,0 +1,366 @@
+      rpc: this.rpc,
+      litNodeClient: this.litNodeClient,
+      network: this.network,
+      processEnvs: this.processEnvs,
+    };
+  }
+
+  /**
+   * Creates a new person with the given name.
+   * @param name - The name of the person.
+   * @returns The newly created person.
+   * @throws Error if the name is not provided.
+   */
+  async createNewPerson(name: string) {
+    console.log('[𐬺🧪 Tinny Environment𐬺] Creating new person:', name);
+    if (!name) {
+      throw new Error('Name is required');
+    }
+    const key = await this.getAvailablePrivateKey();
+    const privateKey = key.privateKey;
+    const envConfig = this.getEnvConfig();
+
+    console.log(
+      '[𐬺🧪 Tinny Environment𐬺] Creating new person with private key:',
+      JSON.stringify(key)
+    );
+    console.log(
+      '[𐬺🧪 Tinny Environment𐬺] Creating new person with envConfig:',
+      envConfig
+    );
+
+    const person = new TinnyPerson({
+      privateKey,
+      envConfig,
+    });
+
+    await person.spawn();
+
+    this.world.set(name, person);
+
+    return person;
+  }
+
+  /**
+   * Retrieves a person from the world by their name.
+   * @param name - The name of the person to retrieve.
+   * @returns The person object if found, or undefined if not found.
+   */
+  getPerson(name: string) {
+    return this.world.get(name);
+  }
+
+  /**
+   * Creates a random person.
+   * @returns A promise that resolves to the created person.
+   */
+  async createRandomPerson() {
+    return await this.createNewPerson('Alice');
+  }
+
+  /**
+   * Sets the global execute JS version for a specific network.
+   * @param network - The network for which to set the execute JS version.
+   * @param version - The execute JS version to set.
+   */
+  setGlobalExecuteJsVersion = (
+    network: LIT_TESTNET,
+    version: LIT_ENDPOINT_VERSION
+  ) => {
+    if (this.processEnvs.NETWORK === network) {
+      process.env[LIT_ENDPOINT.EXECUTE_JS.envName] = version;
+    }
+  };
+
+  /**
+   * Sets the global PKP sign version for the specified network.
+   * @param network - The network for which to set the PKP sign version.
+   * @param version - The PKP sign version to set.
+   */
+  setGlobalPkpSignVersion = (
+    network: LIT_TESTNET,
+    version: LIT_ENDPOINT_VERSION
+  ) => {
+    if (this.processEnvs.NETWORK === network) {
+      process.env[LIT_ENDPOINT.PKP_SIGN.envName] = version;
+    }
+  };
+
+  setUnavailable = (network: LIT_TESTNET) => {
+    if (this.processEnvs.NETWORK === network) {
+      throw new Error('LIT_IGNORE_TEST');
+    }
+  };
+
+  /**
+   * Init
+   */
+  async init() {
+    await this.setupLitNodeClient();
+    await this.setupSuperCapacityDelegationAuthSig();
+  }
+
+  /**
+   * Context: the reason this is created instead of individually is because we can't allocate capacity beyond the global
+   * max capacity.
+   */
+  setupSuperCapacityDelegationAuthSig = async () => {
+    const privateKey = await this.getAvailablePrivateKey();
```

Comment:
Does it not matter which key we use here? If not, why? I'm curious why we're not using a `person`?